src/App.js

```diff
@@ -0,0 +1,13 @@
+import './index.css';
+import Buttons from './components/buttons';
+
+function App() {
+  return (
+    <>
+      <Buttons />
+    </>
+
+  );
+}
+
+export default App;
```

Comment:
- [ ] Great job setting up your react app. However, while trying to delete all the boilerplate, you deleted one core file which your `index.html` file inside public folder. This is the file that makes every component you render get displayed on the browser. Kindly create a folder called `public` in the root directory, create a file called `index.html` inside the folder and put below snippet inside the file.

```
  <!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta
      name="description"
      content="Web site created using create-react-app"
    />
    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
    <title>React App</title>
  </head>
  <body>
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>
  </body>
</html>

```
Then go and delete all other boilerplace like `App.test.js`, and delete them wherever they're called to avoild any error while loading your page.